packages/p2p/src/components/dp2p-blocked/__tests__/dp2p-blocked-description.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Dp2pBlockedDescription from '../dp2p-blocked-description';
+import { useStores } from 'Stores/index';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+describe('<Dp2pBlockedDescription />', () => {
+    it('it should return `P2P transactions are locked. This feature is not available for payment agents.`', () => {
+        (useStores as jest.Mock).mockReturnValue({
+            general_store: {
+                is_p2p_blocked_for_pa: true,
+                is_high_risk: true,
+                is_blocked: false,
+            },
+        });
+        render(<Dp2pBlockedDescription />);
+        expect(
+            screen.getByText('P2P transactions are locked. This feature is not available for payment agents.')
+        ).toBeInTheDocument();
+    });
+
+    it('it should return `To enable this feature you must complete the following:`', () => {
+        (useStores as jest.Mock).mockReturnValue({
+            general_store: {
+                is_p2p_blocked_for_pa: false,
+                is_high_risk: true,
+                is_blocked: false,
+            },
+        });
+        render(<Dp2pBlockedDescription />);
+        expect(screen.getByText('To enable this feature you must complete the following:')).toBeInTheDocument();
+    });
+
+    it('it should return `Please use live chat to contact our Customer Support team for help.`', () => {
+        (useStores as jest.Mock).mockReturnValue({
+            general_store: {
+                is_p2p_blocked_for_pa: false,
+                is_high_risk: false,
+                is_blocked: false,
+            },
+        });
+        render(<Dp2pBlockedDescription />);
+        expect(screen.getByText(/to contact our Customer Support team for help./)).toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.getByText('to contact our Customer Support team for help.')).toBeInTheDocument();
```

Lets change this and using it like the other strings inside the test file

Reply:
it throws error since this one element has sub element inside it. This is how it looks like 
`Please use 
          <span
            class="link link--orange"
          >
            live chat
          </span>
           to contact our Customer Support team for help.`

Reply:
ok then let's keep it like this :+1: